examples/from_snowflake.py

```diff
@@ -0,0 +1,28 @@
+"""Example of using PandasAI with a Snowflake"""
+
+from pandasai import SmartDatalake
+from pandasai.llm import OpenAI
+from pandasai.connectors import SnowFlakeConnector
+
+
+snowflake_connector = SnowFlakeConnector(config={
+        "account": "ehxzojy-ue47135", 
+        "database": "SNOWFLAKE_SAMPLE_DATA",
+        "username": "test",
+        "password": "*****",
+        "table": "lineitem",
+        "warehouse" : "COMPUTE_WH",
+        "dbSchema": "tpch_sf1",
+        "where": [
+            # this is optional and filters the data to
+            # reduce the size of the dataframe
+            ["l_quantity", ">","49"]
+        ],
+    })
+
+OPEN_AI_API = "Your-API-Key"
+llm = OpenAI(api_token=OPEN_AI_API)
+df = SmartDatalake([snowflake_connector], config={"llm": llm})
```

Comment:
Guess since it's a single connector, we can use the `SmartDatframe` here instead, like this:

```
df = SmartDatframe(snowflake_connector, config={"llm": llm})
```